parseInt.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Run a small snippet of JavaScript during a mabl flow/journey
+ * @param {object} - mablInputs Object containing input
+ *                   variables (mablInputs.variables.user)
+ * @param {function} callback - The callback function
+ */
+function mablJavaScriptStep(mablInputs, callback) {
+
+    let element = document.querySelector('REPLACE_WITH_CSS_SELECTOR');
+
+    if (element) {
+        let elementText = element.innerText;
+        let parsed = parseInt(elementText);
```

Comment:
You should always include the radix, i.e., `parseInt(elementText, 10)`. Otherwise, it can accidentally parse things that look like octal or hex values.

Reply:
Will do thank you